src/mon/NVMeofGwMap.cc

```diff
@@ -801,8 +801,14 @@ int NVMeofGwMap::blocklist_gw(
 {
   // find_already_created_gw(gw_id, group_key);
   NvmeGwMonState& gw_map = created_gws[group_key][gw_id];
+  NvmeNonceVector nonces;
+  for (auto& state_itr: gw_map.sm_state) {
+    // to make blocklist on all clusters of the failing GW
+    nonces.insert(nonces.end(), gw_map.nonce_map[state_itr.first].begin(),
+        gw_map.nonce_map[state_itr.first].end());
```

Comment:
I don't understand what's going on here -- wouldn't this blocklist the nonce for every ana group in the gw group, and not just the ones active for that gw?

I'd normally expect to find an explanation for this in the commit message, but "fix blocklist issues" doesn't really explain what this is meant to fix.

Reply:
Initially the monitor's blocklist  blocked only clusters that belong to the same  load balance group  of failed GW.
The namespace load balance broke this concept since namespace being created in ana-group cluster could dynamically migrate to other group. So we do not have records anymore to what clusters belong the IOs of namespaces of failing load-balance (ana) group . We have to blocklist all the clusters. Makes sense?

Reply:
But I have t change to commit description message, you are right

Reply:
Hmm, ok.  Probably best to add a comment explaining that, but sounds reasonable.